cycledash/api/comments.py

```diff
@@ -174,8 +177,30 @@ def get_last_comments(n=5):
         q = select(cols.values()).order_by(
             desc(cols.created)).limit(n)
         comments = [camelcase_dict(dict(c)) for c in con.execute(q).fetchall()]
-    return epochify_comments(comments)
+    return userify_comments(epochify_comments(comments))
 
+def userify_comments(comments):
+    """Given comments with userIds, attaches the relevant user
+    info to the comments
+    """
+    for comment in comments:
+        userify_comment(comment)
+    return comments
+
+def userify_comment(comment):
```

Comment:
I'd call this something like `add_user_to_comment` to make it a little clearer that's going on (I see the parallels with `epochify_comment`, but it's doing something a bit different).